core/arch/arm/plat-imx/conf.mk

```diff
@@ -425,5 +425,8 @@ CFG_IMX_CAAM ?= y
 endif
 endif
 
+CFG_IMX_I2C ?= n
+CFG_IMX_I2C_CLK_RATE ?= 24000000
```

Comment:
Commit "conf: plat-imx: i2c configuration" should come before commit "drivers: imx_i2c: add support for imx6ull" since the later relies on `CFG_IMX_I2C_CLK_RATE`.